web/package.json

```diff
@@ -131,6 +131,7 @@
         "jsep": "1.3.8",
         "jsonpath-plus": "7.2.0",
         "jszip": "3.10.1",
+        "latest": "^0.2.0",
```

Comment:
Why did you add this? It seems like maybe a package that would allow a dev to setup automated checks for new upgrades on a nodejs server, but doesn't seem helpful for us.
Please remove this

Reply:
done